Assets/Scripts/Player/Sword.cs

```diff
@@ -38,7 +42,14 @@ void OnTriggerEnter2D(Collider2D other)
             Debug.Log("Hit bullet");
             GameObject bulletInstance = Instantiate(reflectedBullet, other.gameObject.transform.position, Quaternion.identity);
             bulletInstance.GetComponent<Rigidbody2D>().velocity = -other.gameObject.GetComponent<Rigidbody2D>().velocity;
-            bulletInstance.GetComponent<ReflectedBullet>().Damage = other.gameObject.GetComponent<EnemyBullet>().Damage;
+            if (other.gameObject.GetComponent<EnemyBullet>() != null)
+            {
+                bulletInstance.GetComponent<ReflectedBullet>().Damage = other.gameObject.GetComponent<EnemyBullet>().Damage;
+            }
+            else if (other.gameObject.GetComponent<BossBullet>() != null)
+            {
+                bulletInstance.GetComponent<ReflectedBullet>().Damage = other.gameObject.GetComponent<BossBullet>().BulletDamage;
+            }
```

Comment:
maybe add a final else here, setting the damage to a base number if an attack is reflected that doesn't have a set damage value

Reply:
Made napalm bombs (and regular bombs) no longer get reflected